.kitchen.yml

```diff
@@ -3,18 +3,42 @@ driver:
   name: vagrant
 
 driver_config:
-  provider: vmware_fusion
   customize:
-    memory: 512
+    cpus:   2
+    memory: 2048
+  require_chef_omnibus: true
+  network:
+    - [ 'forwarded_port', { guest: 8080, host: 8080 } ]
 
 provisioner:
   name: chef_solo
+  environments_path: "environments"
+  nodes_path: "nodes"
+  data_bags_path: "data_bags"
+  encrypted_data_bag_secret_key_path: "test/data/secret.txt"
```

Comment:
They get created in the next revision when test kitchen is switched to chef zero.